backend/src/main/java/ch/puzzle/okr/deserializer/DeserializerHelper.java

```diff
@@ -0,0 +1,36 @@
+package ch.puzzle.okr.deserializer;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.IOException;
+import java.util.Map;
+
+@Component
+public class DeserializerHelper {
+
+    public DeserializerHelper() {
+    }
+
+    public <T> T deserializeMetricOrdinal(JsonParser jsonParser, Map<String, Class<? extends T>> map,
```

Comment:
`map` is not a very descriptive name. Maybe change it to something like `validTypes`, something that describes it better.